views/searchWithFilter.njk

```diff
@@ -74,7 +77,6 @@
     )}}
 
     {% if showGroupOrPrisonDropdown %}
-    <div class="govuk-list">
```

Comment:
removed this unnecessary container since want the active case load buttons close to the related search box